### PR TITLE
Update CHANGELOG for release 8.11.0

### PR DESCRIPTION
**Pull request by ivanmjartan**
---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10042/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12011/ : SUCCESS in 8m 25s

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Improved custom attribute filters was added.  We added support for customization of the AttributeFilter and AttributeFilterButton components and new features, such as static and hidden elements.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json:5]: were added

Comment (no23reason) [reply common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json:5]: also, there are two spaces before the second sentece

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Dashboard plugins to customize attribute filters was added. You can now customize the attribute filters with dashboard plugins.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json:5]: Maybe

> Dashobard plugin APIs to...

?

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-02.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Core API for attribute filter components was added. These components now allow the implementation of custom attribute filters.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-02.json:5]: allow implementation (no "the" there)

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json (hunk 1 above)
Comment (ivanmjartan) [reply common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json]: Fixed

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json (hunk 2 above)
Comment (ivanmjartan) [reply common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json:5]: Fixed

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-02.json (hunk 3 above)
Comment (ivanmjartan) [reply common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-02.json:5]: Fixed

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Dashboard plugins APIs to customize attribute filters was added. You can now customize the attribute filters with dashboard plugins.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-01.json:5]: APIs were added

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3070

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**